content/docs/faq-functions.md

```diff
@@ -1,26 +1,26 @@
 ---
 id: faq-functions
-title: Passing Functions to Components
+title: Melewatkan fungsi-fungsi ke dalam komponen
 permalink: docs/faq-functions.html
 layout: docs
 category: FAQ
 ---
 
-### How do I pass an event handler (like onClick) to a component? {#how-do-i-pass-an-event-handler-like-onclick-to-a-component}
+### Bagaimana saya melewatkan event handler (seperti onClick) pada sebuah komponen? {#how-do-i-pass-an-event-handler-like-onclick-to-a-component }
 
-Pass event handlers and other functions as props to child components:
+Anda bisa melakukannya dengan cara meletakkan event handlers dan fungsi-fungsi lain sebagai props untuk child components:
 
 ```jsx
 <button onClick={this.handleClick}>
 ```
 
-If you need to have access to the parent component in the handler, you also need to bind the function to the component instance (see below).
+Jika Anda butuh untuk mengakses parent component pada handler tersebut, maka anada juga harus mem-bind fungsi tersebut pada component instance (akan dibahas di bagian selanjutnya)
 
-### How do I bind a function to a component instance? {#how-do-i-bind-a-function-to-a-component-instance}
+### Bagaimana saya mem-bind fungsi pada sebuah component instance {#how-do-i-bind-a-function-to-a-component-instance}
```

Comment:
```suggestion
### Bagaimana saya mem-_bind_ fungsi pada sebuah _component instance_ {#how-do-i-bind-a-function-to-a-component-instance}
```

Reply:
Thank you @vferdiansyah !!!
I've commit your suggestions.